### DISCUSSION TITLE
Thanks for the implementation

### DISCUSSION THREAD

**Discussion by criosfera** [jxoesneon/IPFS#3] [Show and tell]
I just find out about this and i am very grateful, keep it up please.

I don't know dart, i just use gpt for my code, but believe me, if i could i would help this project.

**Comment by jxoesneon** [comment]
Thank you so much @criosfera! 🙏

Your kind words mean a lot to us. We're thrilled that you found this project useful!

**Great news - we just released v1.2.0!** 🎉

This release includes:
- 🐛 Critical bug fixes for the DHT routing table
- ✨ New DHT value storage operations (`storeValue`, `getValue`)
- 📦 Memory-efficient file streaming with `addFileStream`
- 🧪 423 tests now passing

You can get the latest version from [pub.dev](https://pub.dev/packages/dart_ipfs):

```yaml
dependencies:
  dart_ipfs: ^1.2.0
```

Even without Dart experience, there are many ways to contribute:
- 📝 Documentation improvements
- 🐛 Bug reports
- 💡 Feature suggestions
- ⭐ Star the repo to help others discover it!

Feel free to open issues or discussions if you have questions. Welcome to the community! 🚀
